packages/web/README.md

```diff
@@ -9,43 +7,61 @@ Natura Design System - Web Components
 
 ## Installation
 
-Install the packages in your project directory with:
+Install the packages in your project root directory with:
+
+### npm
 
-```sh
-# with npm
+```shell script
 npm i --save @naturacosmeticos/natds-web
+```
 
-# with yarn
+### yarn
+
+```
 yarn add @naturacosmeticos/natds-web
 ```
 
+### Minimum requirements
+
+* `react` version `16.8.4` or higher (`16.10.2` or higher recommended);
+* `react-dom` version `16.8.4` or higher (`16.10.2` or higher recommended).
+
 ## Usage
 
-```jsx highlight-line="2"
-import { Chip, Provider, themes } from '@naturacosmeticos/natds-web'
+```jsx highlight-line="3"
+import React from "react"
+import { Button, Provider, themes} from "@naturacosmeticos/natds-web"
 
 const { natura } = themes;
 
-return (
-  <Provider theme={natura.light}>
-    <Chip
-      clickable={boolean}
-      disabled={boolean}
-      label={string}
-      onDelete={function}
-    />
-  </Provider>
+export const YourApp = () => (
+   <Provider theme={natura.light}>
+      <Button color={"primary"}>Button component</Button>
+      <Button variant={"contained"}>Contained button</Button>
+   </Provider>
 )
 ```
 
-## About Fonts
+## Is included in package
+
+When you install `@naturacosmeticos/natds-web`, the following packages will be installed too:
+
+- `@material-ui/core`
+- `@naturacosmeticos/natds-icons`
+- `@naturacosmeticos/natds-styles`
+- `@styled-components`
+
+Thus, it is not necessary to install them to use the Design System.
+
+## Not included in package
+
+### Fonts
 
-This package do not provide any fonts, only its font family names. If you desire to use the theme font, you've got to import them on your application by yourself. We recommend taking a look at [Google Fonts](https://fonts.google.com/), since some of our fonts are available for free over there.
+This package does not provide any fonts, only its font family names. If you desire to use the theme font, you've got to import them on your application by yourself. We recommend taking a look at [Google Fonts](https://fonts.google.com/), since some of our fonts are available for free over there.
```

Comment:
Essa observação me parece um aviso importante para quem está instalando o pacote já no início das informações, e poderia ser colocado como uma nota.

Reply:
@eduardogc tem razão, vou alterar isso